src/main/blackbox/blackbox.c

```diff
@@ -1589,12 +1589,15 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_PREFER_BARO, "%d", positionConfig()->altitude_prefer_baro);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_LPF, "%d",         positionConfig()->altitude_lpf);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_D_LPF, "%d",       positionConfig()->altitude_d_lpf);
-        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_HOVER_THROTTLE, "%d",       positionControlConfig()->hover_throttle);
-        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_LANDING_ALTITUDE, "%d",     positionControlConfig()->landing_altitude_m);
-        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_P, "%d",           positionControlConfig()->altitude_P);
-        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_I, "%d",           positionControlConfig()->altitude_I);;
-        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_D, "%d",           positionControlConfig()->altitude_D);;
-        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALTITUDE_F, "%d",           positionControlConfig()->altitude_F);
+
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_HOVER_THROTTLE, "%d",           positionControlConfig()->hover_throttle);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALT_CONTROL_THROTTLE_MIN, "%d", positionControlConfig()->alt_control_throttle_min);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ALT_CONTROL_THROTTLE_MAX, "%d", positionControlConfig()->alt_control_throttle_max);
```

Comment:
- Personal still prefer using just `throttle_min` and `throttle_max` as these are part of position control struct.
- Like `hover_throttle` is not `ALT_CONTROL_HOVER_THROTTLE`

Reply:
@KarateBrot what do you think?
Hover throttle could have non-altitude control uses.
Whereas those min and max values are purely for altitude control, like the pids... hmm perhaps all should have `alt_control_` as their prefix.

Reply:
I also think the alt_control_ prefix is not needed. Both variables are part of the positionConfig (that's already a bit like a prefix) and there is no other min and max throttle in there, so no risk of mixing them up.

Reply:
I almost went ahead and went alt_control in front of all of them after Mark's first comment :-)

Thanks for the input - very difficult.

I think that `hover_throttle` and `landing_altitude` are clear enough on their own.  Most likely we will only need one `hover_throttle` for the craft, and probably only one `landing_altitude`.  They are not directly related to position control functions, even though that's where they most commonly would be used.  So let's set them aside for now, as:

- `hover_throttle`, and
- `landing_altitude`

'Position' as a word suggests 'placement' or 'static position', as in 'position hold', whereas the same PIDs would, most likely, be applicable for GPS Rescue, and that's not really a 'position', it's an autonomous flight mode.  I reflected on this for some time.

I had no problem with `altitude_control` but position control was much harder.  I considered `pos_control`, `position_control`, `yaw_control`, `xy_control` and other options.

In the end, I think that 'automatic' or 'autonomous' control is likely most intuitive.  That might lead to:

- `auto_control_alt_p, i, d, f`
- `auto_control_yaw_p` - yaw control will need a separate value.
- `auto_control_roll_pitch_p, i, d, f`

We can use the pitch:roll slider to boost the value for roll up to pitch; a single set of xy pids should suffice.

For throttle, name like 'throttle_min', alone, in the CLI, doesn't show which parameter group the value belongs to when the user goes `get throttle`.  Without the grouping, the name function is unclear.  Especially as previously the name was `gps_rescue_throttle_min`.

Perhaps a 'generic' control name might work for throttle, eg:
- `throttle_control_min` and 
- `throttle_control_max` ?

That retains the `control` modifier word - though I remain unsure that we really need these throttle limits.

Still unsure what's the best option.

Reply:
- Makes sense to define these longer names as CLI parameters which are mapped to the offset in the PG.
- PG variables only have to be unique within it's structure and as @KarateBrot mentioned we already use `positionControlConfig()->` as prefix.
- Like the idea of renaming `positionControlConfig()->` to `autoPilot()->`

Reply:
It's a lot easier to track a variable through the code if the CLI name and the PG variable name are the same.  That's why I don't like camel-casing the PG name, or using a PG name that's different from the CLI name.

`autoPilot` for the PG config is a bit more fun than positionControlConfig.

I guess we could go 
`auto_pilot_p`
`auto_pilot_yaw_p`
`auto_pilot_altitude_p`

etc...

Reply:
@haslinghuis 
>     * Like the idea of renaming `positionControlConfig()->` to `autoPilot()->`

It needs to have the postfix "Config", eg. `autopilotConfig_t`

Reply:
Done!  - see https://github.com/betaflight/betaflight/pull/13954/commits/1b361d341b6d3fedacb5251e89065fc4d5915b5b